### ISSUE TITLE
Anti-PTPA

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7243]
**Preferred term label:**
Anti-tyrosine phosphatase region of islet antigen-2 antibody positivity

**Synonyms**
Anti-PTPA

**Definition (free text, please give PubMed ID)**
PMID: 32979862

**Parent term (use hpo.jax.org/app)**
under Anti-IA2A

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT  can you clarify if this is a synonym for Anti-islet antigen-2 antibody positivity or a subclass (child)?

**Comment by MaudDLVT** [comment]
@pnrobinson from: PMID: 32979862
"Samples were tested for autoantibodies to [...] the protein tyrosine phosphatase region of islet antigen-2 (PTPA) and the related IA-2βA, while autoantibodies to IA-2A were reassayed using the current harmonized method."
It's an antibody against a part of islet antigen-2 so the terms are not equal. This should be a subclass from my point of view.

**Comment by pnrobinson** [comment]
added
